fedimint-client/src/lib.rs

```diff
@@ -1605,6 +1607,18 @@ pub async fn get_config_from_db(db: &Database) -> Option<ClientConfig> {
     config
 }
 
+pub async fn get_invite_code_from_db(db: &Database) -> Option<InviteCode> {
+    let mut dbtx = db.begin_transaction().await;
+    #[allow(clippy::let_and_return)]
```

Comment:
**suggestion:** I would just apply the suggested clippy fix. Makes sense to me.

Reply:
we need to drop dbtx before other local vars which forces us to use let and return

Reply:
```
pub async fn get_config_from_db(db: &Database) -> Option<ClientConfig> {
    db.begin_transaction()
        .await
        .find_by_prefix(&ClientConfigKeyPrefix)
        .await
        .next()
        .await
        .map(|(_, config)| config)
}
```

Works.